### PR TITLE
[Windows] Add signature validation

### PR DESCRIPTION
**Pull request by sergei-pyshnoi**
# Description

Adding signature validation for several windows packages 

## Check list
- [ ] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by Alexey-Ayupov — DISMISSED** [review]

File: images/win/scripts/Installers/Install-VS.ps1 [hunk 1]
```diff
@@ -4,14 +4,16 @@
 ################################################################################
 
 $toolset = Get-ToolsetContent
+Write-Host $toolset.visualStudio.signature "111"
```
Comment (erik-bershel) [inline images/win/scripts/Installers/Install-VS.ps1:7]: I'm curious what exactly this exact string does?

Comment (sergei-pyshnoi) [reply images/win/scripts/Installers/Install-VS.ps1]: oops. Removed

**Review by Alexey-Ayupov — APPROVED** [review]

**Review by Alexey-Ayupov — APPROVED** [review]

**Pull request merged**